client/my-sites/plan-features-2023-grid/components/test/billing-timeframe.tsx

```diff
@@ -122,6 +124,105 @@ describe( 'PlanFeatures2023GridBillingTimeframe', () => {
 		const discountedPrice = formatCurrency( planPrices.discountedPrice, 'INR', {
 			stripZeros: true,
 		} );
-		expect( container ).toHaveTextContent( `per month, ${ discountedPrice } for the first year` );
+		expect( container ).toHaveTextContent(
+			`per month, ${ discountedPrice } for the first two years`
+		);
+	} );
+
+	test( 'should show full-term discounted price when plan is 3-yearly', () => {
```

Comment:
`Non-blocking / Nit`:

Would you mind describing what full-term discounted means?

Reply:
Sure! "full-term discounted price" refers to the price for the complete duration of the plan with any applicable discounts. The discount could either be a currency-specific discount or a discount due to the application of pro-rated credits. For example:
<img width="286" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/fdc24dbb-c099-43a2-9a9b-95b5d5a097bf">

"280" is the "per month discounted price", and 3360 is the "full-term discounted price".